src/objects/api/fields.py

```diff
@@ -42,7 +44,23 @@ def to_internal_value(self, data):
         try:
             return self.get_queryset().get_by_url(data)
         except ObjectDoesNotExist:
-            self.fail("does_not_exist", value=smart_str(data))
+            # if service is configured, but objec_type is missing
+            # let's try to create an ObjectType
+            service = Service.get_service(data)
+            if not service:
+                self.fail("does_not_exist", value=smart_str(data))
+
+            uuid = get_uuid_from_path(data)
+            object_type = self.get_queryset().model(service=service, uuid=uuid)
+
+            try:
+                object_type.clean()
+            except ValidationError:
+                self.fail("does_not_exist", value=smart_str(data))
```

Comment:
Is the error code a copy paste mistake?

Reply:
No. In `object_type.clean()` we try to request the object type url. If it's not existed `ValidationError` will be raised.